css/styles.css

```diff
@@ -0,0 +1,104 @@
+body {
+  font-family: sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  margin: auto;
+  padding: 20px;
+  background: #ffcc33;
+}
+
+a {
+  font-weight: bold;
+  color: #05080C;
+	}
+
+ul {
+  list-style: square;
+	}
+
+ul p {
+  color: #05080C;
+	}
+
+li {
+	list-style: square;
+  color: #05080C;
+	}
+
+strong {
+  color: #030609;
+  font-style: italic;
+	}
+
+nav {
+  color: #222222;
+  text-align: center;
+	}
+
+
+h1 {
+	font-family: sans-serif;
+	font-size: 100px;
+	letter-spacing: -3px;
+	text-transform: uppercase;
+	text-align: center;
+	margin: 0px;
+	}
+
+th, td {
+    text-align: center;
+    padding: 0px;
+}
+
+
+h3 {
+	font-size: 40px;
+	text-transform: uppercase;
+	letter-spacing: -2px;
+	margin: 0px;
+	color: #030609;
+	text-align: center;
+	}
+
+p {
+	font-family: Georgia, serif;
+	margin-top: 0;
+	margin-bottom: 7px;
+	margin-right: 0px;
+	margin-left: 0px;
+	padding-left: 90px;
+	padding-right: 90px;
+	color: #05080C;
+	text-align: center;
+	}
+
+.p2 {
+	font-family: Georgia, serif;
+	margin-top: 0;
```

Comment:
@dlevkovits Not problem with the code here, but just remember you can consolidate the code with sinlge line code:
```css
margin: 0 0 7px 0;
```
Same thing with the padding.